### ISSUE TITLE
Deepin Calendar not at all useful 

### ISSUE THREAD

**Issue by cacarr-pdxweb** [linuxdeepin/developer-center#1043]
The lack of ability to sync with cloud calendar services like Google Calendar, Apple Calendar, etc, makes this software completely useless. Continued development of it serves no purpose.

**Comment by hualet** [comment]
@shule1987

**Comment by keybreak** [comment]
Please, keep any cloud / sync functional strictly optional in Settings and **disabled** by default (for privacy reasons).

**Comment by lolimay** [comment]
Yes, it really essential if deepin wants to be more open and attractive. It will be very nice if deepin can integrate some popular cloud service into deepin family apps.

**Comment by CareF** [comment]
Strongly agree! 
And Google has pretty good API available to get calendar information. 
It might be even better if deepin can develop a kit of email, calendar, contacts and tasks, similar to evolution-data-server with a deepin style front-end.

**Comment by zefram-cochrane** [comment]
Casting about for an acceptable calendar now. This one is about as useful as a paperweight. Minetime looks good ... but on deepin it gives no notifications! I'm going to go try to flatpak install the gnome calendar ...

**Comment by Darren-Haynes** [comment]
Yes it will be great if the deepin team can make the calendar. I love the deepin desktop but the inability to sync with google drive or even to set alerts is a serious missing feature. Will have to install gnome calendar for now. 
Are there plans to improve the functionality of this calendar?

**Comment by BLumia** [comment]
AFAIK there was plan to add alarms / reminders but we are still don't have enough resources to working on this feature. Consider use another calender app with alarms / reminders support and keep an eye on this project :)

**Comment by sudipmandal** [comment]
I wrote a utility to sync Google Calendar with Deepin Calendar
[https://github.com/sudipmandal/deepin-ext-cal-sync](https://github.com/sudipmandal/deepin-ext-cal-sync)

Download the release and follow the instructions to sync your google calendar with Deepin.

Project is developed in C# dotnetcore and is fully open source, so any C# developers please feel free to pitch in and help enhance it.

Hope it helps :)
